modules/efk-logging-elasticsearch-add-remove.adoc

```diff
@@ -46,13 +46,13 @@ For example, to scale up your cluster:
 [source,yaml]
 ----
 nodes:
-  - nodeCount: 2
+  - nodeCount: 4
     nodeSpec:
       resources: {}
     storage: {}
 ----
 
-. To scale down, create a Cluster Logging Custom Resource (CR) to reduce the number of nodes of a specific type:
+. To scale down, create a Cluster Logging Custom Resource (CR) to reduce the number of nodes:
```

Comment:
@mburke5678  scale down is not supported.

Reply:
@anpingli If the user scales up the number of nodes, can he scale back down to the original number?